microsoft-edge/extensions-chromium/developer-guide/native-messaging.md

```diff
@@ -47,11 +72,13 @@ The following is an example **manifest.json** file:
 }
 ```
 
+---
+
 
 <!-- ====================================================================== -->
 ## Step 2 - Create your native messaging host manifest file
 
```

Comment:
@sonakshimore 
remainder of file - check the usage of "native messaging host manifest file"